ports/windows/variants/manifest.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Spurious newline

Reply:
Could this file also just be left out? Even if it's to enable future expansion, now this just adds noise